### ISSUE TITLE
Possible Data Issue in gerudo_fortress.json

### ISSUE THREAD

**Issue by dcavoulacos** [mracsys/tootr#45]
I was looking through the entrance data for Gerudo Fortress, and I think there might be some mixup in the data between `GF Lower Vines -> Hideout 4 Torch Jail U Tunnel` and `Gerudo Fortress by GTG -> Hideout 4 Torch Jail L Tunnel`:

```
"GF Lower Vines -> Hideout 4 Torch Jail U Tunnel": {
    "area": "Gerudo Fortress",
    "reverse": "Hideout 4 Torch Jail U Tunnel -> GF Lower Vines",
    "oneWay": false,
    "oneWayArea": "",
    "connector": "Hideout 4 Torch Jail L Tunnel -> Gerudo Fortress by GTG",
    "isReverse": false,
    "alias": "1F Near Toll Gate",
    "reverseAlias": "from 1F Near Toll Gate",
    "tag": "4 Torch Jail U Tunnel",
    "tagRep": true,
    "tagRules": [],
    "enableTag": true,
    "type": "hideoutInterior",
    "lKey": "",
    "aLink": "",
    "userALink": "",
    "eLink": "",
    "userELink": "",
    "oneWayELink": "",
    "userOneWayELink": "",
    "shuffled": false
},
"Gerudo Fortress by GTG -> Hideout 4 Torch Jail L Tunnel": {
    "area": "Gerudo Fortress",
    "reverse": "Hideout 4 Torch Jail L Tunnel -> Gerudo Fortress by GTG",
    "oneWay": false,
    "oneWayArea": "",
    "connector": "Hideout 4 Torch Jail U Tunnel -> GF Lower Vines",
    "isReverse": false,
    "alias": "2F North Vines",
    "reverseAlias": "from 2F North Vines",
    "tag": "4 Torch Jail L Tunnel",
    "tagRep": true,
    "tagRules": [],
    "enableTag": true,
    "type": "hideoutInterior",
    "lKey": "",
    "aLink": "",
    "userALink": "",
    "eLink": "",
    "userELink": "",
    "oneWayELink": "",
    "userOneWayELink": "",
    "shuffled": false
}```

At the very least, the aliases are mixed up; `1F Near Toll Gate` should match `Gerudo Fortress by GTG -> Hideout 4 Torch Jail L Tunnel`

**Comment by mracsys** [comment]
You're right, the aliases are backwards. Thanks for reporting it!

**Comment by 3pacgangstarap** [comment]
added a quick fix for this in #46

**Comment by mracsys** [comment]
Merged with #46
